### ISSUE TITLE
Difficult to tell when buttons are disabled

### ISSUE THREAD

**Issue by EugeneZ** [austintackaberry/ydkjs-exercises#166]
It's difficult for me to tell when buttons are disabled. It could be my monitor, but I think it would be a good idea to make a non-clickable button very obvious. Grey is a good color for that, so I propose we change the color of a clickable button.  I started this process in PR #165. [Material design has some nice documentation on buttons.](https://material.io/design/components/buttons.html) Personally, I would recommend against removing disabled buttons because I think it is surprising behavior when users see buttons suddenly appear.

Any suggestions?

**Comment by radotreyes** [comment]
Very good point, I agree that the buttons are pretty ambiguous right now. We're starting to overhaul the design of the site (#115 ), so I think this issue can be taken in as a part of that task; what do you think about adding the task of designing new button assets to the scope of that issue?

**Comment by EugeneZ** [comment]
It's up to you. I personally think it's better to have multiple smaller tasks so that they can be more easily done incrementally. Still, it's good to know that a redesign is in progress. My proposal is to put this issue on hold until a theme is in place, and then come back to this and re-evaluate. It's possible that buttons will come naturally once a theme is in place.
